server/tournaments/index.ts

```diff
@@ -607,6 +607,7 @@ export class Tournament extends Rooms.RoomGame {
 		}
 		this.isBracketInvalidated = true;
 		this.room.add(`|tournament|start|${this.players.length}`);
+		this.modlog('TOUR START', null, `The tournament has started with ${this.players.length} users.`);
```

Comment:
```suggestion
		output.modlog('TOUR START', null, `The tournament has started with ${this.players.length} users.`);
```

Your build error is because your CommandContext is `output`, not `this`.

Reply:
Thanks. Coming from Java I struggle a bit with the CommandContext element of JS. That solved pretty much all the problems that I ran into. Do you think this line of code would be better in the individual start and autostart commands or here?

Reply:
I think it's better to do it here; in general, less repetition is better. It's called "DRY" – "Don't Repeat Yourself", and it's a standard programming principle.

(Also, ideally don't click "Resolve conversation" after asking a question but before getting an answer for it. "Resolve" means that the conversation is finished.)

Reply:
Alright noted about "Resolve conversation". After shortening the modlog entry message, I see no further changes currently.

The code has been tested on a dev server, and it works just fine.